text/0117-unbrick-collective.md

```diff
@@ -0,0 +1,172 @@
+# RFC-0117: The Unbrick Collective
+
+|                 |                                                                                          |
+| --------------- | ---------------------------------------------------------------------------------------- |
+| **Start Date**  | 22 August 2024                                                                           |
+| **Description** | The Unbrick Collective aims to help teams rescuing a para once it stops producing blocks |
+| **Authors**     | Bryan Chen, Pablo Dorado                                                                 |
+
+## Summary
+
+A followup of the [RFC-0014]. This RFC proposes adding a new collective to the Polkadot Collectives
+Chain: The Unbrick Collective, as well as improvements in the mechanisms that will allow teams
+operating paras that had stopped producing blocks to be assisted, in order to restore the production
+of blocks of these paras.
+
+## Motivation
+
+Since the initial launch of Polkadot parachains, there has been many incidients causing parachains
+to stop producing new blocks (therefore, being _bricked_) and many occurrences that requires
+Polkadot governance to update the parachain head state/wasm. This can be due to many reasons range
+from incorrectly registering the initial head state, inability to use sudo key, bad runtime
+migration, bad weight configuration, and bugs in the development of the Polkadot SDK.
+
+Currently, when the para is not unlocked in the _paras registrar_[^1], the `Root` origin is required to
+perform such actions, involving the governance process to invoke this origin, which can be very
+resource expensive for the teams. The long voting and enactment times also could result significant
+damage to the parachain and users.
+
+Finally, other instances of governance that might enact a call using the `Root` origin (like the
+Polkadot Fellowship), due to the nature of their mission, are not fit to carry these kind of tasks.
```

Comment:
This sentence is too hard to parse.  It's they're too busy, right?

Reply:
Not necessarily. It's just a matter of mission and scope. 😅

Same case with the Unbrick Collective: it's scope would be providing assistance to para teams which need help unbricking their para, not helping teams design their newest runtime version, or auditing code (in which case, your suggestion of an auditing collective sounds great).